src/index.html

```diff
@@ -5,8 +5,352 @@
+              class="nav__link"
+              href="#laptops&computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <section
+        class="card content__card card--1"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27` Retina 5K Monoblock (MNED2UA/A)"
+          class="card__image"
+        >
+
+        <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__name">Price:</p>
+
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </section>
+
+      <section class="card content__card card--2">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27` Retina 5K Monoblock (MNED2UA/A)"
+          class="card__image"
+        >
+
+        <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__name">Price:</p>
+
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <a href="#" class="card__button">
+          Buy
+        </a>
+      </section>
+
+      <section class="card content__card card--3">
```

Comment:
Do you really need this selector ?
```suggestion
      <section class="card content__card">
```